README.md

```diff
@@ -1 +1,170 @@
+<p align="center">
+    <img src="logo.png" width="350", height="220">
+</p> 
+<p align="center">A Fast and Smart SpellCorrection using Sound and Edit-distance based Correction available in English and 10 indian languages.      
+</p>  
+<p align="center">  
+  <a href="https://github.com/hellohaptik/spello/stargazers">  
+    <img src="https://img.shields.io/github/stars/hellohaptik/spello.svg?colorA=orange&colorB=orange&logo=github"  
+         alt="GitHub stars">  
+  </a> 
+  <a href="https://pepy.tech/project/spello/">  
+      <img src="https://pepy.tech/badge/spello" alt="Downloads">  
+  </a>   
+  <a href="https://pypi.org/project/spello/">  
+      <img src="https://img.shields.io/pypi/v/spello?colorB=brightgreen" alt="Pypi package">  
+  </a>  
+  <a href="https://github.com/hellohaptik/spello/issues">
+        <img src="https://img.shields.io/github/issues/hellohaptik/spello.svg"
+             alt="GitHub issues">
+  </a>
+  <a href="https://github.com/hellohaptik/spello/blob/master/LICENSE">  
+        <img src="https://img.shields.io/github/license/hellohaptik/spello.svg"  
+             alt="GitHub license">  
+  </a>
+  <a href="https://github.com/hellohaptik/spello/graphs/contributors">  
+        <img src="https://img.shields.io/badge/all_contributors-5-blue.svg"  
+             alt="Contributors">  
+  </a>  
+</p>  
+  
+<p align="center">  
+ <a href="#what-is-it">What is it</a> •  
+  <a href="#-installation">Installation</a> •  
+  <a href="#-️getting-started">Getting Started</a> 
+</p>  
+</p>
+
+<h2 align="center">What is it</h3>  
+  
+**Spello** is a spellcorrection model using power of two model Symspell and Phoneme in backend to get the best possible spelling suggestion for misspelled words in a text. Phoneme Model uses <a href="https://en.wikipedia.org/wiki/Soundex">Soundex</a> algo in background and is responsible to suggest corrections for sound related mistakes in word. Also, we are using a modifield version of<a href="https://github.com/wolfgarbe/SymSpell"> Symspell</a> model to get spell suggestions based on edit-distances. <br>
+Currently, this module is available for **English(en)** and 10 other indian languages which are **Hindi(hi)**,  **Marathi(mr)**,  **Bengali(bn)**,  **Punjabi(pa)**, **Gujarati(gu)**, **Oriya(or)**, **Tamil(ta)**, **Telegu(tl)**, **Kannada(kn)**, **Malyalam(ml)**.  
+  
+<h2 align="center">💾 Installation</h2>  
+<p align="right"><a href="#what-is-it"><sup>▴ Back to top</sup></a></p>
+Install the spello via `pip`
+
+```bash  
+$ pip install spello
+```  
+
+<h2 align="center">⚡ ️Getting Started</h2> 
+<p align="right"><a href="#what-is-it"><sup>▴ Back to top</sup></a></p>
+  
+#### 1. **Model Initialisation**
+Initialise the model for one of the suppored languages. 
+```python  
+>>> from spello.model import SpellCorrectionModel  
+>>> sp = SpellCorrectionModel(language='en')  
+```  
+
+#### 2. Model Training
+Here, you can provide two types of training data to the model
```

Comment:
"You can chose to train model by providing data in one of the following format"

Reply:
Done